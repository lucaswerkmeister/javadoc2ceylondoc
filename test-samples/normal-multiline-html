/**
 * text with <tt>monospaced</tt> and <b>bold</b>
 * and <i>italics</i> and <b><i>both</i></b>
 */
