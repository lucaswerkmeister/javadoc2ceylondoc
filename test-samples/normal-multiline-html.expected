"text with `monospaced` and **bold**
 and *italics* and ***both***"
